src/Nethermind/Nethermind.Evm/BlobGasCalculator.cs

```diff
@@ -85,10 +84,16 @@ static bool FakeExponentialOverflow(UInt256 factor, UInt256 num, UInt256 denomin
             return false;
         }
 
-        return !FakeExponentialOverflow(Eip4844Constants.MinBlobGasPrice, excessBlobGas, Eip4844Constants.BlobGasPriceUpdateFraction, out feePerBlobGas);
+        var denominator = spec?.IsEip7742Enabled ?? false
+            ? Eip7742Constants.BlobGasPriceUpdateFraction * targetBlobCount
```

Comment:
Where does `Eip7742Constants.BlobGasPriceUpdateFraction` come from? The eip doesn't mention updating the rate of change?

Reply:
i think this is a implementation of the PR to the EIP that was eventually closed and separated into another EIP.

Reply:
Ok interesting - but this line is basically the reason why `BlobGasCalculator` now depends on `IReleaseSpec`, and the EIP does state that blob gas fee accounting is the same as before, so this will straight up cause consensus issue.